build.sbt

```diff
@@ -28,7 +28,6 @@ lazy val scoverageSettings = {
   import scoverage.ScoverageKeys
   Seq(
     ScoverageKeys.coverageExcludedPackages := excludedPackages.mkString(";"),
-    ScoverageKeys.coverageMinimum := 80,
```

Comment:
Assuming the coverage fell below 80%. Did the coverage report indicate anywhere we could improve coverage?